### ISSUE TITLE
Selector support for .on()

### ISSUE THREAD

**Issue by madbonkey** [bendc/sprint#30]
I'd like to start a discussion as to whether it could be feasible to add support for passing a selector to .on(), to facilitate event binding to dynamically created DOM elements.

**Comment by mrjjwright** [comment]
Yes, and the data object param.  :)  But if we don't get them because of some performance reason, no biggie for me.  Just a request.

**Comment by mrjjwright** [comment]
Oh nevermind, this is trivial with a closure.
